apps/api/src/app/change/usecases/get-changes/get-changes.usecase.ts

```diff
@@ -95,11 +95,19 @@ export class GetChanges {
     entityId: string,
     environmentId: string
   ): Promise<IViewEntity | Record<string, unknown>> {
-    const item = await this.notificationTemplateRepository.findOne({
+    let item = await this.notificationTemplateRepository.findOne({
       _environmentId: environmentId,
       _id: entityId,
     });
 
+    if (!item) {
```

Comment:
Could we maybe add an API e2e for this one?

Reply:
I worked on it when you wrote this ;)

Reply:
I added an expect for changes as well now :)